Sebastian-Sole/handicappin
# ✨ feat: Exceptional round

## What was done
- Added support for exceptional round scores in accordance to [USGA and WHS](https://www.usga.org/content/usga/home-page/handicapping/world-handicap-system/topics/exceptional-score-reduction.html)
- Added `exceptionalRoundAdjustment` column to Rounds table in Supabase
- Cleaned up old code

## How to Test
- Submit an exceptional round (score diff 7 under handicap)
- See that previous rounds have had score differentials adjusted accordingly

## Future Work
- Round router clean up
- Display exceptional round in dashboard and round calculation
